doc/user_guide/model.rst

```diff
@@ -1034,6 +1046,74 @@ on the ``centre`` parameter.
              sigma |  True | 0.00999184 | 2.68064163 |       None |       None
             centre |  True | 9.99980788 | 2.68064070 |        7.0 |       14.0
 
+.. _linear_fitting-label:
+
+Linear least squares
+^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 1.7
+
+Linear fitting can be used to address some of the drawbacks of non-linear optimization:
+
+- it doesn't suffer from the *starting parameters* issue, which can sometimes be problematic
+  with nonlinear fitting. Since linear fitting uses linear algebra to find the
+  solution (find the parameter values of the model), the solution is a unique solution,
+  while nonlinear optimization uses an iterative approach and therefore relies
+  on the initial values of the parameters.
+- it is fast, because i) in favorable situations, the signal can be fitted in a vectorized
+  fashion, `i. e.` the signal is fitted in a single run instead of iterating over
+  the navigation dimension; ii) it is not iterative, `i. e.` it does the
+  calculation only one time instead of 10-100 iterations, depending on fast
+  the non-linear optimizer will converge.
+
+However, linear fitting can *only* fit linear models and will not be able to fit
+parameter which varies *non-linearly*.
+
+A component is considered linear when its free parameters scale the component only 
+in the y-axis. For the exemplary function ``y = a*x**b``, ``a`` is a linear parameter, whilst ``b`` 
+is not. If ``b.free = False``, then the component is linear. 
+Components can also be made up of several linear parts. For instance, 
+the 2D-polynomial ``y = a*x**2+b*y**2+c*x+d*y+e`` is entirely linear.
+
+.. note::
+
+    After creating a model with values for the nonlinear parameters, a quick way to set 
+    all nonlinear parameters to be ``free = False`` is to use ``m.set_parameters_not_free(only_nonlinear=True)``
+
+If all components in a model are linear, then a linear optimizer can be used to
```

Comment:
Is there a list of the components + parameters somewhere, which shows which ones are linear, and which ones are not?